src/pages/_document.tsx

```diff
@@ -0,0 +1,40 @@
+import { DocumentHeadTags, documentGetInitialProps } from '@mui/material-nextjs/v13-pagesRouter';
+import type { DocumentHeadTagsProps } from '@mui/material-nextjs/v13-pagesRouter';
+import { DocumentContext, DocumentProps, Head, Html, Main, NextScript } from 'next/document';
+
+export default function MyDocument(props: DocumentProps & DocumentHeadTagsProps) {
```

Comment:
wrong `&`

Reply:
![Uploading Screen Shot 2024-03-27 at 17.52.33.png…]()

Reply:
<img width="1157" alt="Screen Shot 2024-03-27 at 17 52 33" src="https://github.com/defi-wonderland/web3-react-boilerplate/assets/162849030/bb0a98c4-9ec4-4669-9571-250b6e985cac">

Reply:
wow it breaks the gh styling